README.md

```diff
@@ -35,8 +36,12 @@ potential for reuse. Is this "situational" character which precludes them to be
 offered as 'off-the-shelf' functionality by solution providers, and therefore
 creates the need for a tool like WireCloud.
 
-This project is part of [FIWARE](http://www.fiware.org). Check it out in the [Catalogue](http://catalogue.fiware.org/enablers/application-mashup-wirecloud)
+This project is part of [FIWARE](https://www.fiware.org/). For more information
+check the FIWARE Catalogue entry for
+[Context Processing, Analysis and Visualization](https://github.com/Fiware/catalogue/tree/master/processing).
 
+| :books: [Documentation](https://wirecloud.rtfd.io/) | :page_facing_up: [Site](https://conwet.fi.upm.es/wirecloud) | :mortar_board: [Academy](https://fiware-academy.readthedocs.io/en/latest/processing/wirecloud) | :whale: [Docker Hub](https://hub.docker.com/r/fiware/wirecloud/) | 
```

Comment:
@mjimenezganan, @rafitanba, I think Jason has done a great job. The only thing we need to analyse is the use of the https://conwet.fi.upm.es/wirecloud link as, in my opinion, has to be redone from scratch (current version is missing valid https certificates, the content is very very outdated, ...). We can merge Jason's Pull Request and change this latter, but I want to know your opinion because this has to be updated ASAP.